### DISCUSSION TITLE
＋6️⃣ Buckaroo recurring payments

### DISCUSSION THREAD

**Discussion by remcotolsma** [pronamic/wp-pronamic-pay#184] [Ideas]
@rvdsteege posted on **May 27, 2016**:

> Buckaroo heeft ook iets met recurring, maar als ik het goed lees moet je dan nog wel zelf het mandaat bijhouden/meesturen » https://support.buckaroo.nl/index.php/Service_SEPA_Direct_Debit

_Internal Basecamp comment:_ https://basecamp.com/1810084/projects/10966871/todos/254387016#comment_422457002

Unfortunately, the Buckaroo page mentioned is no longer available at the moment.

This feature has been requested multiple times:

**May 12, '20, 11:07**

> Ik zag dat de abonnementen functie alleen via Mollie mogelijk is. Nu zag ik dat Buckaroo ook terugkerende betalingen ondersteund. Staat support voor deze recurring payments op jullie roadmap of helemaal niet?

_Internal HelpScout ticket:_ https://secure.helpscout.net/conversation/1162693371/

**Feb 21, '20, 10:35**

> We hebben jullie plugin momenteel op een website van een van onze klanten draaien, ze werken momenteel met Mollie en de klant wilt overstappen naar Buckaroo. Er wordt gebruik gemaakt van normale en recurring payments, kunnen jullie ons vertellen of de Buckaroo koppeling ook de mogelijkheid heeft tot recurring payments?

_Internal HelpScout ticket:_ https://secure.helpscout.net/conversation/1090051111/19237/

**Jun 4, '18, 15:05**

> Voor een klant willen we graag een Woocommerce webshop met terugkerende betalingen koppelen aan Buckaroo als payment gateway. Nu zie ik op jullie site dat er inderdaad een Ideal koppeling mogelijk is met Buckaroo maar ik kan niet zien of dit ook geldt voor terugkerende betalingen met automatische incasso. We hebben jullie plugin al eens eerder gebruikt voor een klant die een soortgelijke opzet heeft qua webshop (Woocommerce + Woocommerce Subscriptions + Pronamic Ideal) maar die gebruiken Mollie als Payment gateway.
> 
> Echter voor deze klant is het noodzakelijk dat we Buckaroo gebruiken vanwege hun accreditatie bij Currence B.V. en de rechtsgeldigheid van digitale machtigingen die via hun platform worden gedaan.
> 
> Kunnen jullie mij vertellen of de Pronamic Ideal plugin ook een Ideal integratie heeft voor Buckaroo met recurring payment? (soortgelijk aan die van Mollie)

_Internal HelpScout ticket:_ https://secure.helpscout.net/conversation/594307134/14756/

**Jan 31, '17, 14:28**

> Voor onze site gebruiken wij uw plug-in samen met Gravity Forms. Onze payment provider is Buckaroo. Voor iDeal betalingen werkt alles prima, maar wij zouden graag de machtigingen die binnenkomen van onze donatie pagina (eenmalig of periodiek), willen automatiseren. Uw plug-in voorziet daar nog niet in. Is er op korte termijn een oplossing voor te verwachten?

_Internal HelpScout ticket:_ https://secure.helpscout.net/conversation/376464815/1160/

**Jan 5, '17, 14:01**

> Kan het kloppen dat machtiging niet wordt ondersteund in de backaroo -> gravity forms plugin? Ik kan het nergens vinden, maar weet ook niet zeker of ik iets over het hoofd zie.

_Internal HelpScout ticket:_ https://secure.helpscout.net/conversation/376473355/1384/

**Comment by remcotolsma** [comment]
New request:

**2021-08-17T10:17:49+02:00**

> Geachte heer of mevrouw, Voor een klant van ons online marketing bureau willen we abonnementen verkopen via Buckaroo. Met Pronamic zijn abonnementen via Mollie nu wel mogelijk, mijn vraag is dan ook of er plannen zijn om dit voor Buckaroo mogelijk te maken op korte termijn.
> Hoor het graag.
> Alvast bedankt!

_Internal HelpScout ticket:_ https://secure.helpscout.net/conversation/1603584860/22536?folderId=1425710

**Comment by LeoOosterloo** [comment]
Also requested by mail

**2021-08-19**
Klant vraagt om abonnements betalingen met Buckaroo. Is dat goed te doen met Pronamic Pay? Zijn daarin betalingsschema's (termijnen, bedragen, first payments) aan te maken, ook voor Buckaroo?
Groet

**Comment by remcotolsma** [comment]
New request:

> Hello,
> 
> I was wondering why there is only support for Mollie on recurring payments in gravity forms, my client has a website that Mollie denied so we have to use Buckeroo now, which has recurring payments.
> Is it going to be supported later maby?
> Is there a workaround we can use?
> 
> Love to hear from you!

_Internal HelpScout ticket:_ https://secure.helpscout.net/conversation/1857471989/23864/

**Comment by rvdsteege** [comment]
Also requested in https://freescout.pronamic.nl/conversation/10111?folder_id=25

> Wij zouden graag ook periodieke donaties aanbieden, maar als ik het goed begrijp ondersteunt Pronamic op dit moment alleen Mollie. Hebben jullie plannen om ook periodieke donaties via Buckaroo te ondersteunen, en zo ja op welke termijn?

**Comment by rvdsteege** [comment]
Also requested in https://freescout.pronamic.nl/conversation/11650?folder_id=25

> Voor een klant ben ik aan het overschakelen van Mollie naar Buckaroo omdat zij als stichting opeens abonnementsgeld aan Mollie moeten gaan betalen. Na een zoektocht zijn ze bij Buckaroo uitgekomen. Ik heb het formulier gemaakt, via Pronamic Pay Gravity Forms aan Buckaroo gekoppeld. Volgens mij alles gedaan zoals zou moeten maar bij een testbetaling krijg ik nu terug dat de betalingsgateway geen recurring payments support. Nu weet ik dat Buckaroo dat wel ondersteunt. KLeine zoektocht via Cloude krijg ik terug dat dit in Pronamic Pay niet wordt ondersteund. OP een andere website heb ik het wel in combinatie met Mollie gedaan. Hopelijk kunnen jullie mij verder helpen. Mollie is door deze best wel hoge abonnementsvorm die ze moeten afnemen geen optie. Het gaat om een non-profit stichting.
